packages/drawer/src/views/Drawer.tsx

```diff
@@ -601,35 +610,59 @@ export default class DrawerView extends React.PureComponent<Props> {
               importantForAccessibility={open ? 'no-hide-descendants' : 'auto'}
               style={styles.content}
             >
-              {renderSceneContent({ progress: this.progress })}
+              {renderSceneContent({
+                // make 'sidebar' initially visible on the big screen to avoid annoying animation
+                progress: this.bigScreenSidebar
+                  ? new Animated.Value(1)
```

Comment:
nit: lets move this to top of the scope to avoid creating this animated value on render